presto-main/src/main/java/com/facebook/presto/server/ResourceGroupStateInfoResource.java

```diff
@@ -68,6 +68,27 @@ public ResourceGroupInfo getQueryStateInfos(@PathParam("resourceGroupId") String
         throw new WebApplicationException(NOT_FOUND);
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Encoded
+    @Path("getStaticResourceGroupInfo/{resourceGroupId: .+}")
```

Comment:
do we envision this endpoint to stay longer, if yes then i would say let's make it little little more generic, i.e. by taking a key=value as input where isStatic=true and use that instead of dedicating the entire endpoing to pull static resource group.

Reply:
Makes sense I will change it. We should be able to receive the whole tree if needed.